sklearn/linear_model/_logistic.py

```diff
@@ -1609,6 +1597,28 @@ class LogisticRegressionCV(LogisticRegression, LinearClassifierMixin, BaseEstima
     0.98...
     """
 
+    _parameter_constraints = {**LogisticRegression._parameter_constraints}
+
+    _parameters_to_remove = ["C", "warm_start", "l1_ratio"]
```

Comment:
do really want to keep this variable in the model even after updating the dictionary.
I mean the following would the same job:

```python
for param in ["C", "warm_start", "l1_ratio"]:
    _parameter_constraints.pop(param)
```

Reply:
I have removed the `_parameters_to_remove` variable